operations/iuf/workflows/management_rollout.md

```diff
@@ -171,6 +171,20 @@ or `Management_Storage`. These values will be needed when upgrading NCN storage
         iuf -a "${ACTIVITY_NAME}" run -r management-nodes-rollout --limit-management-rollout Management_Master
         ```
 
+    1. Use `kubectl` to remove the `iuf-prevent-rollout=true` label from `ncn-m002`.
+
+        (`ncn-m001#`) Remove label from `ncn-m002`.
+
+        ```bash
+        kubectl label nodes "ncn-m002" --overwrite iuf-prevent-rollout-
```

Comment:
Is the dash needed after rollout in, --overwrite iuf-prevent-rollout**-**

Reply:
Yes, that's the syntax used to clear a label.